### ISSUE TITLE
FFTW libraries not found by externally built remote module on Windows

### ISSUE THREAD

**Issue by dzenanz** [InsightSoftwareConsortium/ITK#1922]
Trying to build current master ([eece13d](https://github.com/InsightSoftwareConsortium/ITKMontage/commit/eece13df670e17da002a1f4a20497c8c059eff99)) of ITKMontage by pointing to current master (67eb78ed6d7643717af33d8441997ef28d86ea9b) of ITK results in link errors.

```text
6>------ Build started: Project: MontageTestDriver, Configuration: RelWithDebInfo x64 ------
6>LINK : fatal error LNK1104: cannot open file 'fftw3_threads.lib'
```
The library itself exists:
<img width="470" alt="Screenshot 2020-07-13 14 37 42" src="https://user-images.githubusercontent.com/1792121/87342020-85a08d00-c518-11ea-83ed-7a599fb416aa.png">

### Environment

Windows 10 x64, VS2019, ITK itself compiled with v140 toolset, CMake 3.17.3. I don't think that version details are important, just being Windows is probably enough. I didn't try to reproduce it on other systems.

**Comment by hjmjohnson** [comment]
Perhaps we should restore the previous cmake behavior:

if(win32)
  message(FATAL_ERROR "FFTW internal build not supported on windows, set ITK_USE_SYSTEM_FFTW")
endif()

**Comment by SimonRit** [comment]
RTK nightly builds fail on Windows since the last update of FFTW even if it uses externally built FFTW, see [here](https://my.cdash.org/index.php?project=RTK&date=2020-07-10). Should I change my build code to use the new variable names, i.e., `FFTWD_LIBRARIES` and `FFTWF_LIBRARIES` instead of `FFTWD_LIB` and `FFTWF_LIB`?

**Comment by SimonRit** [comment]
Solving #1930 did not solve externally built RTK, see, e.g., [here](https://my.cdash.org/viewBuildError.php?buildid=1859250).

**Comment by dzenanz** [comment]
I still get link errors on Windows, as of 85cb1d5860f81cbb3f35606b71e4cf73188837bd from yesterday.

**Comment by SimonRit** [comment]
I think I have fixed this issue for RTK, see  SimonRit/RTK#407. Don't hesitate to comment there if you agree (or not) with the solution!
